src/pages/Information/index.js

```diff
@@ -0,0 +1 @@
+export { default } from "./Information";
```

Comment:
Couldn't Information.jsx be written in this index since it's just double exporting at this point?

Reply:
The export file was Aaron's suggestion on my file structure.